docs/modules/n1ql/partials/n1ql-rest-api/admin/definitions.adoc

```diff
@@ -697,6 +712,41 @@ The <<queryN1qlFeatCtrl,cluster-level>> `queryN1qlFeatCtrl` setting specifies th
 When you change the cluster-level setting, the node-level setting is over-written for all nodes in the cluster. +
 **Default** : `76` +
 **Example** : `"0x1"`|integer (int32)
+|**node-quota** +
+__optional__|[#node-quota]
+Sets the soft memory limit for this node, in MB.
+The garbage collector tries to keep below this target.
+It is not a hard, absolute limit, and memory usage may exceed this value.
+
+When set to `0` (the default), there is no soft memory limit.
+
+The <<queryNodeQuota,cluster-level>> `queryNodeQuota` setting specifies this property for the whole cluster.
+When you change the cluster-level setting, the node-level setting is over-written for all nodes in the cluster. +
+**Default** : `0`|integer (int32)
+|**node-quota-val-percent** +
+__optional__|[#node-quota-val-percent]
+The percentage of the `node-quota` that is dedicated to tracked value content memory across all active requests on this node.
+(The `memory-quota` setting specifies the maximum amount of memory an individual request may use on this node.)
```

Comment:
Can we add the word "document" before "memory" here - since we struggle the fact quota isn't absolute in Query. ("... maximum amount of document memory an individual...")

Reply:
Done